04-onion-routing.md

```diff
@@ -1007,15 +1252,27 @@ the decrypted byte stream.
 The complete amount of the multi-part payment was not received within a
 reasonable time.
 
+1. type: BADONION|PERM|24 (`invalid_onion_blinding`)
+2. data:
+   * [`sha256`:`sha256_of_onion`]
+
+The onion sent to a blinded path didn't match the recipient's encrypted relay requirements.
```

Comment:
Well, it's more "Some error occurred within the blinded path".

Reply:
Updated in https://github.com/lightning/bolts/pull/765/commits/bdbf694e5874b6931db3668e22e3178b249ad06d